js/src/util.js

```diff
@@ -100,7 +67,7 @@ const Util = (($) => {
 
   const Util = {
 
-    TRANSITION_END: 'bsTransitionEnd',
+    TRANSITION_END: transition.end,
```

Comment:
I'll find a solution 😉
and that's better to do it now instead of when we will have a stable release because we're still in beta so breaking change can happen and must happen now